plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -389,11 +430,25 @@ def assignUserToGroup(self, user, group_identifier, access_level):
     def updateUser(self, user, arguments, uncheckable_args):
         changed = False
 
+        arguments = sanitize_arguments(arguments)
```

Comment:
Does this make sense? Everything in `arguments` has a `dict` as a value, so this should be a no-op.